1. values added: 20
2. final result: 20
3. values added: 20
4. (returns error) the variable result was declared inside the if using keyword let, so it errors when trying to access outside of the block scope since the variable result doesn't exist outside of the block.
5. (returns error) cannot change the value of a const value once assigned
6. (returns error) previous executed code returns an error so this will not run, however even if previous code didn't error, the variable result is out of block scope and wouldn't be accessible